packages/components/src/input/src/Input.css

```diff
@@ -1,32 +1,32 @@
 .o-ui {
-    --o-ui-input-padding: var(--o-ui-sp-3);
```

Comment:
normal error and focus:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/d2ab6aee-7950-4703-af32-ca99611b37d8)
error and focus when tabbing and you have a plugin like keeper:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/795b1767-f9b8-4c06-8bcb-cb7aef821c29)

Reply:
We can't really focus on all the third party extensions. If anything I expect that the tab focus on the eye before anything else.

Reply:
What he says.

Reply:
I've fixed the focus rendering that corresponds to @vicky-comeau's second example.